code/modules/SCP/SCPs/SCP-294.dm

```diff
@@ -8,67 +8,64 @@ GLOBAL_LIST_EMPTY(scp294_reagents)
 	layer = 2.9
 	anchored = TRUE
 	density = TRUE
-	var/uses_left = 12
+	var/list/banned_chems = list(
+	/datum/reagent/adminordrazine,
+	/datum/reagent/nitroglycerin,
+	/datum/reagent/glycerol,
+	/datum/reagent/coolant,
+	/datum/reagent/frostoil,
+	/datum/reagent/zombie
+	)
+	var/uses_left = 5
 	var/last_use = 0
 	var/restocking_timer = 0
 	SCP = /datum/scp/scp_294
 	var/list/player_names = list()
 	var/player_attack
 	var/mob/living/carbon/victim
 
-/obj/machinery/scp294/Initialize(atom/holder)
-	//get the names of all players at roundstart to save on cpu.
-	//yes this may lead to names not working if they join midround but it adds to the mystery of its 'randomness' lol
-
-	player_names += GLOB.player_list
-	return ..()
-
-/obj/machinery/scp294/Destroy()
-	player_names = null
-	victim = null
-	return ..()
-
```

Comment:
Sometimes I wonder if the people who first coded these SCPs weren't retarded. Why would you get the player list "to save on CPU"?!??? It's literally a global variable, accessing it should be stupid easy! And like hell initializing one 294 will be a CPU hog!